packages/atomic/src/components/common/pager/pager-buttons.spec.ts

```diff
@@ -0,0 +1,191 @@
+import {
+  pagerNextButton,
+  pagerPageButton,
+  pagerPageButtons,
+  pagerPreviousButton,
+} from './pager-buttons';
+
+describe('pagerButtons', () => {
+  let container: HTMLElement;
+  let i18n: I18n;
+
+  describe('pagerPreviousButton', () => {
+    beforeEach(async () => {
+      i18n = await createTestI18n();
+      container = document.createElement('div');
+      document.body.appendChild(container);
+
+      render(
+        html`${pagerPreviousButton({
+          props: {
+            i18n,
+            icon: ArrowLeftIcon,
+          },
+        })}`,
+        container
+      );
+    });
+
+    afterEach(() => {
+      document.body.removeChild(container);
+    });
+
+    test('should render the button with the correct attributes', () => {
+      const button = container.querySelector('button');
+      expect(button).toHaveAttribute('aria-label', 'Previous');
+      expect(button).toHaveClass('btn-outline-primary');
+      expect(button).toHaveAttribute('part', 'previous-button');
+    });
+
+    test('should render the icon with the correct attributes', () => {
+      const icon = container.querySelector('atomic-icon');
+      expect(icon).toHaveAttribute('part', 'previous-button-icon');
+    });
+  });
+
+  describe('pagerNextButton', () => {
+    beforeEach(async () => {
+      i18n = await createTestI18n();
+      container = document.createElement('div');
+      document.body.appendChild(container);
+      render(
+        html`${pagerNextButton({
+          props: {
+            i18n,
+            icon: ArrowRightIcon,
+          },
+        })}`,
+        container
+      );
+    });
+
+    afterEach(() => {
+      document.body.removeChild(container);
+    });
+
+    test('should render the button with the correct attributes', () => {
+      const button = container.querySelector('button');
+      expect(button).toHaveAttribute('aria-label', 'Next');
+      expect(button).toHaveClass('btn-outline-primary');
+      expect(button).toHaveAttribute('part', 'next-button');
+    });
+
+    test('should render the icon with the correct attributes', () => {
+      const icon = container.querySelector('atomic-icon');
+      expect(icon).toHaveClass('w-5');
+    });
+  });
+
+  describe('pagerPageButton', () => {
+    beforeEach(async () => {
+      i18n = await createTestI18n();
+      container = document.createElement('div');
+      document.body.appendChild(container);
+    });
+
+    afterEach(() => {
+      document.body.removeChild(container);
+    });
+
+    test('should render the button with the correct attributes', () => {
+      render(
+        html`${pagerPageButton({
+          props: {
+            groupName: 'pager',
+            page: 1,
+            isSelected: false,
+            text: '1',
+          },
+        })}`,
+        container
+      );
+
+      const input = container.querySelector('input');
+      expect(input).toHaveAttribute('aria-label', '1');
+      expect(input).toHaveAttribute('type', 'radio');
```

Comment:
This does not come from a prop we're passing to the functional component so I don't think we should check it here.